python/lsst/daf/base/dateTime/dateTimeContinued.py

```diff
@@ -19,11 +19,11 @@
 # You should have received a copy of the LSST License Statement and
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
-#
-
 
 __all__ = []
 
+import astropy.time
```

Comment:
I would be inclined to put the astropy.time import inside `toAstropy` because it's a very large package and nothing below daf_base is importing it. In the context of calling this from pipe_tasks or something that doesn't really matter (since astropy will be imported by daf_butler) but thinking of daf_base on its own it would be better, I think, to treat astropy as an optional dependency.